src/scenes/BROWSER/distinctTargets.ts

```diff
@@ -0,0 +1,34 @@
+import { SceneFlexItem, SceneQueryRunner } from '@grafana/scenes';
+import { DataSourceRef } from '@grafana/schema';
+
+import { ExplorablePanel } from 'scenes/ExplorablePanel';
+
+function getQueryRunner(metrics: DataSourceRef) {
+  return new SceneQueryRunner({
+    datasource: metrics,
+    queries: [
+      {
+        expr: `count by (job, target) (
+          count by (url, method) (
+            probe_browser_web_vital_fcp{probe=~"\${probe}", job="\${job}", instance="\${instance}"}
```

Comment:
So taking into account the available metrics we have, grouping by the url on a web vitals metric seemed to me like the best option to distinguish targets.

With this implementation, we'll show 2 targets when a browser check visits two distinct urls:
![image](https://github.com/user-attachments/assets/08a30c5f-e221-486b-909e-e4a7c85d74f6)
![image](https://github.com/user-attachments/assets/e0135cb5-d038-4492-aacc-34dfab3de9fe)

If the browser check only goes to one, we'll just show one target. But I understand this leaves out information about the different interactions a user might have within a page. 

If we think this is not useful information or might be misleading, there's also the option of removing this panel. What do you think? We could also explore the logs to see if there's any other relevant information that we can obtain regarding targets, instead of just relying on the metrics data for this panel.

Regarding the `label` in the query, I removed it as you correctly pointed out it's not a present in the metric's response.